### PR TITLE
[ADD] Cassandra backend for workload dashboard

### PR DESCRIPTION
**Pull request by bhaskarhc**
### What this pr ?
- Cassandra Connection using "cassandra-driver" npm package .
- Database Query to read and write user data in Cassandra .
- [add] -Deployment Yaml link added in yaml.js .

### Connection successful : 
```

|Cassandra-Jiva Connected . !
|__> keyspace `maya` created .
|___> Table `person` created .

```
### Read & Write Query:
screenshot from 2018-11-12 15-18-58

 

Signed-off-by: bhaskarjb <hcbhaskar7@gmail.com>

### CODE REVIEW

File: database/cassandra-jiva/dbconnection.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Uuid = require('cassandra-driver').types.Uuid;
+const cassandra = require('cassandra-driver');
+const PlainTextAuthProvider = cassandra.auth.PlainTextAuthProvider;
+const client = new cassandra.Client({ contactPoints:['localhost:9042'],authProvider: new PlainTextAuthProvider('cassandra', 'cassandra')});
```
Comment (inyee786) [inline database/cassandra-jiva/dbconnection.js]: localhost:9042 ??

File: database/cassandra-jiva/dbconnection.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+client.connect(function(err) {
+    if (!err) {
+        console.log("|Cassandra Connected . !");
+        var query = "CREATE KEYSPACE IF NOT EXISTS maya WITH replication = {'class':'SimpleStrategy', 'replication_factor' : 3};"
+        client.execute(query,function(err) {
+        if (!err) {
+            console.log("|__> keyspace `maya` created .");
```
Comment (inyee786) [inline database/cassandra-jiva/dbconnection.js:12]: give more generic log everywhere

**Comment by inyee786** [comment]
deploy the application first

**Review by chandankumar4 — APPROVED** [review]

**Comment by chandankumar4** [comment]
Need to send this PR again!!

**Pull request closed without merging**